specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json

```diff
@@ -1210,6 +1210,13 @@
             "type": "string"
           }
         },
+        "applicableScenarios": {
+          "description": "The list of scenarios applicable to this metadata entity.",
+          "type": "array",
+          "items": {
+            "type": "string"
```

Comment:
This should be an enum so the SDK can generate strongly typed values for it.

Reply:
Made changes. I will make changes to other PR once this looks good to you.